Makefile

```diff
@@ -27,7 +27,7 @@ clean:
 .PHONY: deps
 deps: clean
 	( cd ./queue/testdata; $(TTCTL) rocks install queue 1.3.0 )
-	( cd ./crud/testdata; $(TTCTL) rocks install crud 1.0.0 )
+	( cd ./crud/testdata; $(TTCTL) rocks install crud 1.1.1 )
```

Comment:
Mb add a changelog entry about it.

Reply:
We don't have a release since crud support is added. So we don't need a record in the changelog.